### DISCUSSION TITLE
Direct connection settings page not shown for HMIP-BBL

### DISCUSSION THREAD

**Discussion by btuerk89** [OpenCCU/OpenCCU#2626] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

After (maybe also before but not checked) firmware update of HmIP-BBL to V1.10.16  the page "Home page > Programs and connections > Direct device connections > Profile setting" is blank. The connection is still working but not editable anymore. Restart of the RaspberryMatic does not solve the issue.

### Describe the behavior you expected

Page should be shown.

### Steps to reproduce the issue

1. Login to RaspberryMatic
2. Navigate to "Home page > Programs and connections > Direct device connections > Profile setting" for the HMIP-BBL
3. Page leaves blank

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Not sure what can be helpful for this issue, but have some logs available or can create new ones.
```

### Additional information

-

**Comment by jens-maus** [comment]
Please open up the Browser Console and see if there is any Error in the Error console display of your browser. In addition, please have a look at the filesystem on the RaspberryMatic if there are any cgi error files in `/usr/local/tmp` directory when you try to edit the device settings.

**Comment by btuerk89** [comment]
Hello @jens-maus 

in Browser (Chrome 32bit latest Version) I could not find any error in console. 

In /usr/local/tmp I found some files. One for each try.
Content: 
```
Return-Path: <CGI script - do not reply>
From:  <CGI script - do not reply>
To: root
Subject:  CGI problem

CGI environment:
REQUEST_METHOD: POST
SCRIPT_NAME: /config/ic_setprofiles.cgi
HTTP_USER_AGENT: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
HTTP_HOST: 192.168.178.[XYZ]
REMOTE_ADDR: 192.168.178.[XYZ]
cgi.tcl version: 1.10.0
input:
iface=HmIP-RF&sender_address=00135D89B55[XYZ]%3A1&receiver_address=00135D89B55[XYZ]%3A4&_=
cookie:
errorInfo:
can't read "ps(LONG_SLATS_MOVEMENTS_TO_SKIP)": no such element in array
    while executing
"cgi_quote_html $ps($param_id)"
    (procedure "cmd_link_paramset2" line 75)
    invoked from within
"cmd_link_paramset2 $iface $address ps_descr ps "LINK" ${special_input_id}_$prn"
    (procedure "set_htmlParams" line 40)
    invoked from within
"set_htmlParams $iface $receiver_address receiver_ps ps_descr_receiver "receiver" $dev_descr_sender(TYPE)"
    (procedure "put_profile_body" line 171)
    invoked from within
"put_profile_body "SENDER""
    (procedure "put_page" line 47)
    invoked from within
"put_page"
```

BR

**Comment by ghost** [comment]
**Neustart der CCU3 hilft**

**Comment by btuerk89** [comment]
Hi @andrzejgilas,

nein, hilft leider nicht. 
Habe mehrfach neu gestartet und versucht den Fehler in den Logs zu finden, bevor ich den Issue hier eröffnet habe. 

BR

**Comment by Baxxy13** [comment]
Was für eine DV ist das denn?
Selbst angelegt oder eine der beiden "Standard-DV's" Taster <--> Aktor?

Erzeuge mal selbst eine DV (virt-Taste <--> Aktor) und gucke ob das Problem da auch auftritt.

**Comment by btuerk89** [comment, marked as answer]
Hi @Baxxy13, 

es waren die beiden Standard-DVs, die nicht mehr gingen. 
Virtuelle Taste als DV mit Aktor ging wie gewohnt anzulegen und zu bearbeiten. 

Ich war dann mal mutig und habe die Standard-DVs gelöscht und neu angelegt.
Hat zum Glück funktioniert und alles ist wieder wie gehabt. 👍 

@jens-maus 
Ich denke der markierte Parameter kam mit der neuen Firmware dazu und da hat sich irgendwas gefressen. 
![grafik](https://github.com/jens-maus/RaspberryMatic/assets/124655843/c95fd5d5-1d18-41c0-8533-2c2e8f7a35c0)
Da es von meiner Seite aus wieder funktioniert, kann der Issue geschlossen werden.

Danke an alle.

BR

**Reply by jens-maus** [reply]
Ok, hab das Issue dann mal in einen Diskussionbeitrag verwandelt damit da ggf. auch andere Leidensgenossen dann drüber stolpern und einen Hinweis haben wie man vorgehen kann um das Problem ggf. zu lösen.

Ob man hierfür einen Workaround/Fix in der WebUI platziert oder machen sollte damit es schon gar nicht zu der Exception kommt bin ich mir ehrlich gesagt noch nicht so sicher…

**Reply by Baxxy13** [reply]
Da die Lösung des Problems einfach zu bewerkstelligen ist denke ich nicht das wir dafür nen Fix brauchen.
Der Aufwand für Test's wäre m.E. zu groß (Up/Downgrade) der Geräte-Firmware.

**Comment by Baxxy13** [comment]
Sehr gut. 
Das löschen und neu anlegen der original DV's hätte ich dann als nächstes empfohlen. 😉
